core/templates/dev/head/directives.js

```diff
@@ -156,3 +156,17 @@ oppia.directive('mobileFriendlyTooltip', ['$timeout', function($timeout) {
     }
   };
 }]);
+
+oppia.directive('updateLanguage', ['$rootScope', function($rootScope) {
```

Comment:
@sean I had tried ti use <html ... lang=<[currentLang]> ...> , but it didnt work properly so i had to write a directive .